apps/govquests-frontend/src/utils/api.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Quest } from "../types/quest";
+
+const API_BASE_URL = "http://localhost:3001";
+
+export async function getQuests(): Promise<Quest[]> {
+  try {
+    const response = await fetch(`${API_BASE_URL}/quests`);
+    if (!response.ok) {
```

Comment:
We will have a couple of repeated code. Take a look into https://kentcdodds.com/blog/replace-axios-with-a-simple-custom-fetch-wrapper to create a handler for fetchs